spec/serializer/MapFishPrintV2VectorSerializer.spec.ts

```diff
@@ -194,20 +196,23 @@ describe('MapFishPrintV2VectorSerializer', () => {
         [829437.9003438022, 6708897.081846992]
       ]])
     });
+
+    const features: Feature<Geometry>[] = [
+      pointFeature1,
+      pointFeature2,
+      lineFeature,
+      polygonFeature
+    ];
+
     const source = new OlSourceVector({
-      features: [
-        pointFeature1,
-        pointFeature2,
-        lineFeature,
-        polygonFeature
-      ]
+      features
     });
     const layerName = 'Vector Layer';
     const layer = new OlLayerVector({
       source: source,
       style: style,
       name: layerName
-    });
+    } as any);
```

Comment:
Why is `any` needed here?

Reply:
Because name is not a valid property here (it is dynamically added to the properties obtainable via `getProperties()` or `get('name')`)..